### PR TITLE
Cleanup Existing OEPs

### LINKED ISSUE #45: OEP code should be in Python
Currently the functions examples are written in C# (see OEP4, OEP5, etc). Since Python is the recommended language for use, these examples should be offered in Python instead.

### PR DESCRIPTION
**Pull request by WyattMufson**
This PR is meant to cleanup the OEP4, OEP5 and OEP8 proposals. 

- It changes all of the code from C# to Python, resolves #45. 
- It cleans up the english in each of the proposals, making them clearer and improving readability.
- Standardizes the spacing and formatting of each proposal.
- Updates the example contracts to be using the version 2 compiler.

### CODE REVIEW

**Comment by WyattMufson** [comment]
@Honglei-Cong @javajoker

File: OEPS/OEP-8.mediawiki [hunk 1]
```diff
@@ -8,129 +8,167 @@
...
+def approve(from, to, tokenId, amount)
 </pre>
-The approve method allows to grant the amount <code>value</code> of Crypto Item <code>tokenId</code> from <code>from</code> account to <code>to</code> account.
-If this function is called again it overwrites tokenId or value.
-The Parameters <code>from</code> and <code>to</code> SHOULD be 20-byte addresses. If not, this method SHOULD throw an exception.
+
+The approve method allows to grant the <code>amount</code> of Crypto Items with the id <code>tokenId</code> from the <code>from</code> address to the <code>to</code> address.
+If this function is called again it overwrites the <code>amount</code> allowed for the <code>to</code> address.
+The parameters <code>from</code> and <code>to</code> must be 20-byte addresses.
+If not, this method must throw an exception.
```
Comment (Honglei-Cong) [inline OEPS/OEP-8.mediawiki]: \<code\>throw\</code\>

File: OEPS/OEP-8.mediawiki [hunk 2]
```diff
@@ -8,129 +8,167 @@
...
+  tokenId: <TOKEN ID>,
+  amount: <AMOUNT>
 }
 </pre>
 
-The approveMulti method allows to grant multiple amount <code>Value</code> of Crypto Item <code>TokenId</code> from <code>From</code> account to <code>To</code> account.
-The Parameters <code>From</code> and <code>To</code> SHOULD be 20-byte addresses. If not, this method SHOULD throw an exception.
+The approveMulti method allows to grant multiple <code>amount</code> of Crypto Items with the id <code>TokenId</code>.
+The Parameters <code>from</code> and <code>to</code> must be 20-byte addresses.
+If not, this method must throw an exception.
```
Comment (Honglei-Cong) [inline OEPS/OEP-8.mediawiki]: format of 'throw'

File: OEPS/OEP-8.mediawiki [hunk 3]
```diff
@@ -8,129 +8,167 @@
...
+  tokenId: <TOKEN ID>,
+  amount: <AMOUNT>
 }
 </pre>
-The transferFromMulti method is used for a withdraw workflow, allowing multiple <code>Spender</code> to withdraw <code>Value</code> of Crypto Item <code>TokenId</code>  from <code>From</code> account to <code>To</code> account.
-The Parameters <code>Spender</code> <code>From</code> and <code>To</code> SHOULD be 20-byte addresses. If not, this method SHOULD throw an exception.
+
+The transferFromMulti method is used for a withdraw workflow - allowing multiple calls to <code>transferFrom</code>.
+The parameters <code>spender</code>, <code>from</code> and <code>to</code> must be 20-byte addresses.
+If not, this method must throw an exception.
```
Comment (Honglei-Cong) [inline OEPS/OEP-8.mediawiki]: ditto

**Pull request merged**